lighthouse-core/lib/web-inspector.js

```diff
@@ -239,10 +239,19 @@ module.exports = (function() {
    * Creates a new WebInspector NetworkManager using a mocked Target.
    * @return {!WebInspector.NetworkManager}
    */
-  WebInspector.NetworkManager.createWithFakeTarget = function() {
+  WebInspector.NetworkManager.createWithFakeTarget = function(driver) {
     // Mocked-up WebInspector Target for NetworkManager
     const fakeNetworkAgent = {
-      enable() {}
+      enable() {},
+      getResponseBody(requestId, onComplete) {
```

Comment:
yah. i think we'd end up with a helper method in driver otherwise.. and it seems slightly more convenient to call a method on the record itself rather than `driver.requestContent(networkRecord).then(..`

so i'm supportive of mocking the real API here, yeah. :)